### PR TITLE
Replace all k/website in localization guide with kubernetes/website

### PR DESCRIPTION
**Pull request by bishal7679**
mentioned issue #34656 

Description :- all mistakes are fixed by this commit in localization.md file.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@bishal7679 please see https://github.com/kubernetes/website/pull/34657#issuecomment-1168228363:
> If the changes are in the same file, I'd request you to incorporate them in another commit to [this PR](https://github.com/kubernetes/website/pull/34657).

**Comment by bishal7679** [comment]
@jihoon-seo Actually I have made that first PR from website not via cloning.
then how can I make another commits into that same PR.
Can you please check out this PR also https://github.com/kubernetes/website/pull/34658 fixed all the mistakes in that file.
Should I close this PR?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/en/docs/contribute/localization.md
Comment (Sea-n) [inline content/en/docs/contribute/localization.md:240]: I think it should be:
```markdown
After adding the language-specific `OWNERS` file, update the
[root `OWNERS_ALIASES`](https://git.k8s.io/website/OWNERS_ALIASES) file with the new Kubernetes teams
```

File: content/en/docs/contribute/localization.md
Comment (Sea-n) [inline content/en/docs/contribute/localization.md:281]: The `k/xxx` abbr is acceptable. But it's not bad to expand it to `kubernetes/xxx`.

File: content/en/docs/contribute/localization.md
Comment (bishal7679) [reply content/en/docs/contribute/localization.md:240]: Then I have to close this PR and make another new for this?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable

File: content/en/docs/contribute/localization.md
Comment (bishal7679) [reply content/en/docs/contribute/localization.md:281]: I think it will increase the readability

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Readable

File: content/en/docs/contribute/localization.md
Comment (bishal7679) [reply content/en/docs/contribute/localization.md:240]: If anyone clicked on [root `OWNERS_ALIASES`] it will redirect into `owners_aliases` docs and there root is not mentioned anywhere. So this `root` should be out of that link.

**Comment by sftim** [comment]
/hold

We should update the PR title. This PR does _not_ fix all the mistakes in the docs, and I'm loth to merge a PR with such a potentially misleading title.

OK to unhold once that title is edited to be more clear.

**Comment by sftim** [comment]
Because of the hold
/approve cancel

It's also OK to re-approve this once the title is sorted

**Comment by bishal7679** [comment]
@sftim PR title is changed now. Can you review it again?

File: content/en/docs/contribute/localization.md
Comment (sftim) [inline content/en/docs/contribute/localization.md:421]: We could make this reference into a second hyperlink.

**Comment by sftim** [comment]
/retitle Replace all k/website in localization guide with kubernetes/website

File: content/en/docs/contribute/localization.md
Comment (bishal7679) [reply content/en/docs/contribute/localization.md:281]: I don't think it's an improvement. It's just to increase readability and saving user's one click.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Readable

**Comment by Sea-n** [comment]
In reply to [this comment](https://github.com/kubernetes/website/pull/34658#discussion_r908402790), I don't agree with "Changing `k/website` to `kubernetes/website` would saving user's click so it's an improvement."

In Slack, PR description, and many places we will use abbr like `k/website` without any explanation or links. It's good for contributors to learn what it mean (if they are not already know) while reading docs.

Anyway, I am not strongly opposite with this PR too. Happy to see LGTM from anyone.

**Comment by Sea-n** [comment]
BTW, I don't quite understand about "mentioned issue #34656" in your PR description, could you explain more about that?

**Comment by bishal7679** [comment]
Actually I didn't make another issue for this PR. And this PR was regarding the mistakes in localization.md file.So I mentioned #34656 issue.
You can avoid this #34656 .
Sorry for that It was my bad.

**Comment by bishal7679** [comment]
@Sea-n Can You recheck again?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: content/en/docs/contribute/localization.md
Comment (Sea-n) [inline content/en/docs/contribute/localization.md:239]: The link syntax shouldn't across lines.

**Review by Sea-n — CHANGES_REQUESTED** [review]
Still not fixed.

File: content/en/docs/contribute/localization.md
Comment (bishal7679) [reply content/en/docs/contribute/localization.md:239]: Okay

**Comment by kbhawkey** [comment]
I am closing this PR. The changes from this PR are now found in https://github.com/kubernetes/website/pull/34847.
/close

**Pull request closed without merging**
